GameServer/spells/Valkyrie/PowerRendSpellHandler.cs

```diff
@@ -60,22 +60,13 @@ public override void OnDirectEffect(GameLiving target)
 
 				if (targetIsGameplayer)
 				{
-					powerRendValue = (int)(target.MaxMana * Spell.Value * GetVariance());
+					powerRendValue = (int)(target.MaxMana * (Spell.Value/100));
```

Comment:
this works for power trap, where we deal % power damage, but might be a problem for valks. if we need to support them later